.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,26 @@
+<!--
+    As we are working towards a stable version of uuid, we require that you 
+    open an issue, before submitting a pull request. If the pull request is 
+    imcomplete, prepend the Title with WIP: 
+-->
+
+**I'm submitting a ...**
```

Comment:
nitpick: This might read a bit better if we change the checkboxes to be singular. So instead of _bug fixes_ we'd say _bug fix_. What do you think?

Reply:
fixed